src/c_api/c_api.cc

```diff
@@ -809,11 +822,10 @@ int MXLoadLib(const char *path) {
       regOp.set_attr<mxnet::FInferShape>("FInferShape", DefaultSubgraphOpShape, plevel);
       regOp.set_attr<FInferStorageType>("FInferStorageType",
                                         DefaultSubgraphOpStorageType, plevel);
-      regOp.set_attr<FResourceRequest>("FResourceRequest",
-                                       DefaultSubgraphOpResourceRequest, plevel);
       regOp.set_attr<nnvm::FMutateInputs>("FMutateInputs",
                                           DefaultSubgraphOpMutableInputs, plevel);
     }
+    regOp.set_attr<FResourceRequest>("FResourceRequest", resc_req, plevel);
```

Comment:
can we put this after line 798 instead? its kinda weird hanging out here in the middle

Reply:
put it to 806 for readability